src/_includes/docs/add-to-app/ios-project/embed-cocoapods.md

```diff
@@ -35,15 +35,16 @@ If you have a different directory structure,
 adjust the relative paths.
 The example directory structure resembles the following:
 
-```plaintext
-/path/to/MyApp
```

Comment:
This makes me realize maybe it would be nice to support titles for the block, similar to how we support them for code blocks.

Do you think supporting one as a parameter to `FileTree` makes sense or if we need it, with our own downstream wrapper div?